flink-state-backends/flink-statebackend-rocksdb/src/main/java/org/apache/flink/contrib/streaming/state/RocksDBKeyedStateBackend.java

```diff
@@ -1484,7 +612,7 @@ private void restoreInstanceDirectoryFromPath(Path source) throws IOException {
 	/**
 	 * Creates a column family handle for use with a k/v state.
 	 */
-	private ColumnFamilyHandle createColumnFamily(String stateName) {
+	public static ColumnFamilyHandle createColumnFamily(String stateName, ColumnFamilyOptions columnOptions, RocksDB db) {
```

Comment:
We could move this to `RocksDBOperationUtils`, now that it became effectively a static helper.